CHANGELOG.md

```diff
@@ -6,6 +6,24 @@ _**Note:** This is in reverse chronological order, so newer entries are added to
 Swift 5.5
 ---------
 
+* Type name is no longer allowed as an argument to a parameter that expects a metatype type e.g.
```

Comment:
"Type names are no longer allowed as an argument to a subscript parameter that expects a metatype type"

Reply:
Adjusted!